compiler/extccomp.nim

```diff
@@ -27,7 +27,9 @@ type
     hasGcGuard,               # CC supports GC_GUARD to keep stack roots
     hasGnuAsm,                # CC's asm uses the absurd GNU assembler syntax
     hasNakedDeclspec,         # CC has __declspec(naked)
-    hasNakedAttribute         # CC has __attribute__((naked))
+    hasNakedAttribute,        # CC has __attribute__((naked))
```

Comment:
So I should just make it `hasAttribute` and `hasDeclspec` which includes compatibility for both naked and noreturn implicitly?